addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -303,6 +303,7 @@ function factory(dependencies) {
             if (this.env.messaging.currentPartner.id === partner_id) {
                 Object.assign(updateData, {
                     message_unread_counter: 0,
+                    pendingSeenMessageId: undefined,
```

Comment:
A bit above on this method, `lastFetchedMessage` is linked to `lastMessage` in the existing code, why? Fetched status seems unrelated to seen. (eg. could have already fetched a more recent message but not seen it)

Reply:
Indeed this seems weird